pkg/state/helmx.go

```diff
@@ -31,6 +31,8 @@ func (st *HelmState) PrepareChartify(helm helmexec.Interface, release *ReleaseSp
 
 	opts.ChartVersion = release.Version
 
+	opts.Namespace = release.Namespace
```

Comment:
Thanks for your contribution!

FYI, this specific line is added via #1300 for fixing #1299, which is certainly the primary goal of this pull request.

I'll review other parts of this shortly!